### PR TITLE
🐛 Fix removing menu pdf

### PR DESCRIPTION
**Pull request by m0ppers**
first version was quickly made up but a big hack. after much googling and trying out stuff this finally looks like a railsy solution (first time ever rails :S)

please feedback if this is idiomatic

### CODE REVIEW

**Comment by m0ppers** [comment]
fixes https://github.com/railslove/recover-backlog/issues/8

**Comment by m0ppers** [comment]
added some simple tests...please check :thinking:

File: spec/models/company_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        expect(company.menu_pdf_link).to be_a(String)
+      end
+    end
+
+    it 'should delete the menu when requested' do
+      company.menu_pdf.attach(io: File.open(__FILE__), filename: 'menu.pdf', content_type: 'application/pdf')
+      expect(company.menu_pdf.blank?).to be false
+      company.update!({:remove_menu_pdf => '1'})
+      expect(company.menu_pdf.blank?).to be true
+      expect(company.menu_pdf_link).to be_nil
```
Comment (salimhb) [inline spec/models/company_spec.rb:23]: You can use the [change matcher](https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/change-matcher)
```suggestion
      expect { company.update!({:remove_menu_pdf => '1'}) }.to change { company.menu_pdf.blank? }.from(false).to(true)
      expect { company.update!({:remove_menu_pdf => '1'}) }.to change { company.menu_pdf_link }.to(nil)
```

**Review by salimhb — CHANGES_REQUESTED** [review]

**Review by salimhb — APPROVED** [review]

**Pull request merged**